applications/StructuralMechanicsApplication/python_scripts/trilinos_convergence_criteria_factory.py

```diff
@@ -49,5 +49,8 @@ def __init__(self, convergence_criterion_parameters):
             Residual = TrilinosApplication.TrilinosResidualCriteria(R_RT, R_AT)
             Residual.SetEchoLevel(echo_level)
             self.mechanical_convergence_criterion = TrilinosApplication.TrilinosOrCriteria(Residual, Displacement)
+            
+        else:
+            raise NameError("WARNING: Not convergence criterion available. The possibilities are: \ndisplacement_criterion \nresidual_criterion \nnd_criterion \nor_criterion")
```

Comment:
can you please make it consistent with the [non-trilinos version](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/convergence_criteria_factory.py#L58)?